tests/test_models.py

```diff
@@ -8,34 +12,74 @@
                           BertForTokenClassification, BertTokenizerFast, DetrForSegmentation, DetrFeatureExtractor,
                           DetrForObjectDetection)
 
-TOKEN = '<INSERT-TOKEN-MECHANISM>'
+SEED = 1337
+random.seed(SEED)
+np.random.seed(SEED)
+torch.manual_seed(SEED)
 
 
 class MyTestCase(unittest.TestCase):
+    project: dl.Project = None
+    package: dl.Package = None
 
-    def setUp(self) -> None:
+    @classmethod
+    def setUpClass(cls) -> None:
+        if torch.cuda.is_available():
+            torch.cuda.manual_seed_all(SEED)
         if dl.token_expired():
-            dl.login_token(TOKEN)
-        self.project = dl.projects.create("hugging_face_adapter_tests")
-        self.package = package_creation(self.project, "../model_adapter.py")
+            dl.login()
+        try:
+            cls.project = dl.projects.create("hugging_face_adapter_tests")
+        except dl.exceptions.Conflict:
```

Comment:
The exception returned from trying to create a project with an existing name is dl.exceptions.InternalServerError.
Please check if the exception is caught